docs/src/reference-guide/authentication/index.md

```diff
@@ -20,11 +20,10 @@ Please enter your password:
 
 After successfully entering your wallet password you should be presented with your new JWT token.
 
-
 ```text
 ✅ Successfully authenticated at https://indexer.fuel.network:29987/api/auth/signature.
 
 Token: eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiODNlNjhiOTFmNDhjYWM4M....
 ```
 
-Use this token in your `Authorization` headers when making requests for operations such as uploading indexers, stopping indexers, and other operations that mutate state in this way.
+Use this token in your `Authorization` headers when making requests for operations such as uploading indexers, stopping indexers, and other operations that mutate state in this way. Further, users can just pass this JWT token value to the `--auth` flag, if using `forc index` commands that support authentication (e.g., `forc index deploy --auth $MY_JWT_TOKEN`).
```

Comment:
Very Small nit. But I think the text might be more readable to break this into to small paragraphs.  We can also remove the connecting word if we do this 'Further,'.  Neither is right or wrong, the thought just occurred to me.

"Use this token in your `Authorization` headers when making requests for operations such as uploading indexers, stopping indexers, and other operations that mutate state in this way.

Users can just pass this JWT token value to the `--auth` flag, if using `forc index` commands that support authentication (e.g., `forc index deploy --auth $MY_JWT_TOKEN`)."